CONTRIBUTING.md

```diff
@@ -1,5 +1,56 @@
-# Contribution Guide
+# How to Contribute to CodecademyCommunity.github.io
```

Comment:
Let's change this to "How to Contribute to the Codecademy Community website" to future-proof it.